images/server/install-packages.sh

```diff
@@ -74,7 +74,6 @@ get_sig_samba_repo() {
     fi
 }
 
-# shellcheck disable=SC2120
```

Comment:
Does this mean that it's expected that all calls to this function will explicitly provide an argument ? if so, I think we should also remove the `if [[ -z $1 ]]; then`.

Reply:
Ah, ok. Thanks.